tools/nl/embeddings/data/preindex/medium/sv_descriptions.csv

```diff
@@ -6330,7 +6330,7 @@ dc/topic/WorkCommute,Work Commute;Work commute;commute mode;journey to work;trav
 dc/topic/YearlyProjectedTemperatureHighs,Projected max temperature for the year;maximum temperature projected year;temperature highs for the year;yearly projected max temperature
 dc/topic/YearlyProjectedTemperatureLows,Projected min temperature for the year;minimum temperature projected year;temperature lows for the year;yearly projected min temperature
 dc/topic/sdg,Sustainable Development Goals;sdg;sdg Sustainable Development Goals;sustainable development goal Sustainable Development Goals
-dc/topic/sdg_1,SDG No Poverty;sdg 1;sdg No Poverty;sdg1;sustainable development goal No Poverty
+dc/topic/sdg_1,No Poverty;poverty;sdg 1;sdg No Poverty;sdg1;sustainable development goal No Poverty
```

Comment:
looks like the sdg topics in current main and sdg have some slight diffs. Do they matter? @pradh